### ISSUE TITLE
using PhyloTreeLib for cutting the tree to obtain viral genera, subfamilies and families

### ISSUE THREAD

**Issue by hlwCoBi** [shiraz-shah/VFCs#2]
Hello! I am currently trialling your code for cutting the tree to obtain viral genera, subfamilies and families. However, my vOTUs.VOCs.tsv, vOTUs.VFCs.tsv, vOTUs.VOCs.tsv, vOTUs.subfamilies.tsv and vOTUs.subfamilies.tsv are all 0kb.
Any insight on how to fix this? Thanks!!

**Comment by hlwCoBi** [comment]
Each time I run treetool.py, I generate a directory containing clusterinfo.txt. The first column of the file is the ID of the sequence, and the second column is the number.

**Comment by shiraz-shah** [comment]
Yes, those clusterinfo files are exactly what you need here. Rename that
file to vOTUs.subfamilies.tsv etc.
